store/timer_layer.go

```diff
@@ -3722,6 +3736,22 @@ func (s *TimerLayerPluginStore) SaveOrUpdate(keyVal *model.PluginKeyValue) (*mod
 	return resultVar0, resultVar1
 }
 
+func (s *TimerLayerPluginStore) SetWithOptions(pluginId string, key string, value interface{}, options model.PluginKVSetOptions) (bool, *model.AppError) {
```

Comment:
Is this related to the File Info store cache?

Reply:
Same as before.